### PR TITLE
Update perlfaq4.pod to use fc for safe lowercase comparisons

### PR DESCRIPTION
**Pull request by briandfoy**

### CODE REVIEW

**Review by bigpresh — COMMENTED** [review]
Would it be worth a brief explanation on why you'd use `fc` (case folding) rather than `lc`?

e.g. something like:

"Note - you'll often see code which uses `lc` instead, e.g. `lc $a cmp lc $b` - whilst that will work, it can be tripped up by certain Unicode characters, and `fc()`, added in 5.16, should be preferred."

and probably also a note that `fc()` is only available if you've explicitly enabled it with `feature`, or you've said `use v5.16;` or higher?  Otherwise, someone following this who only said `use strict` or is updating legacy code with no version dep will just get tripped up.   e.g. the note from `fc`'s entry in `perlfunc`:

> fc is available only if the "fc" feature is enabled or if it is prefixed with CORE::. The "fc" feature is enabled automatically with a use v5.16 (or higher) declaration in the current scope.

**Comment by davorg** [comment]
New version looks good to me. I say ship it.

**Review by davorg — DISMISSED** [review]

**Review by bigpresh — DISMISSED** [review]

**Comment by davorg** [comment]
Are we expecting the Travis checks to complete? It looks like I don't have permission to skip them and merge.

File: lib/perlfaq4.pod [hunk 1]
```diff
@@ -2197,10 +2197,9 @@ such as C<each()>.
 (contributed by brian d foy)
 
 To sort a hash, start with the keys. In this example, we give the list of
-keys to the sort function which then compares them ASCIIbetically (which
-might be affected by your locale settings). The output list has the keys
-in ASCIIbetical order. Once we have the keys, we can go through them to
-create a report which lists the keys in ASCIIbetical order.
+keys to the sort function which then compares them strings. The output list
```
Comment (Grinnz) [inline lib/perlfaq4.pod:2200]: "compares them strings" -> "compares them as strings"

Comment (Grinnz) [reply lib/perlfaq4.pod:2200]: Note that without "use locale", the comparison order may be better described as ASCIIbetical, and not how one would expect locale-based string comparisons. Not that this answer necessarily needs to detail that.

**Comment by karenetheridge** [comment]
thanks, just released!

**Pull request merged**